run_dir/static/js/project_samples.js

```diff
@@ -393,6 +393,10 @@ function load_all_udfs(){
         }
       }
 
+      else if (prettify(key) == 'all_samples_destroyed'){
+       $('#all_samples_destroyed').text(value);
+      }
+
```

Comment:
Why was this needed?

Reply:
What?? That's weird, I removed the `rawdate` class from the HTML-file, since the date only showed up for aborted projects if I kept it. I thought that was the "fix"-commit 😆 
Very weird...I already had the lines from the JS-file, I never removed them, obviously...